easypy/gevent_non_patched_threadpool.py

```diff
@@ -0,0 +1,36 @@
+from logging import getLogger
+from queue import Queue
+import threading
+
+
+_logger = getLogger()
+non_patched_threadpool = None
+
+
+class NonPatchedThreadPool:
```

Comment:
This can't be a standalone module. 
Either move this into gevent.py, or make a `gevent` sub-package with this and more inside...